src/components/Navbar.js

```diff
@@ -134,12 +143,16 @@ class BounceNavbar extends Component {
         if (this.state.goToExplore) {
             // Stub: direct page to my Explore
         }
+        if (this.state.goToCreateClub) {
+            // Stub: direct page to create club
```

Comment:
`pageRedirect = <Redirect to='/create-club'></Redirect>;`

Reply:
Also, it seems like we actually do not need this, because the NavBarSignIn component takes a href field that we can use... Please look into this and ignore my previous my previous comment if it the case.